<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabletAreaSelectionView</class>
 <widget class="QWidget" name="TabletAreaSelectionView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="controlsLayout">
     <item>
      <widget class="QGroupBox" name="screenGroupBox">
       <property name="title">
        <string>Screen</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="screenCenterLayout">
          <item>
           <widget class="Wacom::AreaSelectionWidget" name="screenArea" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="screenToggleButton">
          <property name="text">
           <string>Toggle Screen</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="controlsGroupBox">
       <property name="title">
        <string>Map Screen To</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="fullTabletRadioButton">
          <property name="text">
           <string>Full Tablet</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="tabletAreaRadioButton">
          <property name="text">
           <string>Tablet Area</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="forceProportionsButton">
          <property name="text">
           <string>Set Screen Proportions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calibrateButton">
          <property name="text">
           <string>Calibrate</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="controlsFreeSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="tabletGroupBox">
     <property name="title">
      <string>Tablet</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="areaCenterLayout">
        <item>
         <widget class="Wacom::AreaSelectionWidget" name="areaWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="warningLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Wacom::AreaSelectionWidget</class>
   <extends>QWidget</extends>
   <header>areaselectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>calibrateButton</sender>
   <signal>clicked()</signal>
   <receiver>TabletAreaSelectionView</receiver>
   <slot>onCalibrateClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forceProportionsButton</sender>
   <signal>clicked()</signal>
   <receiver>TabletAreaSelectionView</receiver>
   <slot>onForceProportionsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fullTabletRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>TabletAreaSelectionView</receiver>
   <slot>onFullTabletSelected(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabletAreaRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>TabletAreaSelectionView</receiver>
   <slot>onTabletAreaSelected(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>screenToggleButton</sender>
   <signal>clicked()</signal>
   <receiver>TabletAreaSelectionView</receiver>
   <slot>onScreenToggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCalibrateClicked()</slot>
  <slot>onForceProportionsClicked()</slot>
  <slot>onFullTabletSelected(bool)</slot>
  <slot>onTabletAreaSelected(bool)</slot>
  <slot>onScreenToggle()</slot>
 </slots>
</ui>
